exchanges/stream/buffer/buffer.go

```diff
@@ -200,6 +200,12 @@ func (w *Orderbook) processObUpdate(o *orderbookHolder, u *Update) error {
 		return o.updateByIDAndAction(u)
 	}
 	o.updateByPrice(u)
+	if u.ChecksumFn != nil {
+		err := u.ChecksumFn(o.ob.Retrieve(), u.Checksum)
```

Comment:
Not a change request, but since its relevant, on error of checksum, nothing really happens https://trello.com/c/P9LdDMpE/118-websocket-orderbook-checksum-recovery-via-resubscribe